tests/access.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test, expect } from '@playwright/test';
+import login, { logout } from '@lib/auth';
+import { config } from '@lib/config';
+import {
+  navigateToKafkaList,
+  deleteKafkaInstance,
+  createKafkaInstance,
+  navigateToAccess,
+  grantManageAccess,
+  findAccessRow,
+  revokeAccess
+} from '@lib/kafka';
+
+const testInstanceName = config.instanceName;
+
+test.beforeEach(async ({ page }) => {
+  await login(page);
+
+  await navigateToKafkaList(page);
+
+  await expect(page.getByRole('button', { name: 'Create Kafka instance' })).toBeVisible();
+
+  if ((await page.getByText(testInstanceName).count()) > 0 && (await page.locator('tr').count()) === 2) {
+    // Test instance present, nothing to do!
+  } else {
+    await page.waitForSelector('[role=progressbar]', {
+      state: 'detached',
+      timeout: config.kafkaInstanceCreationTimeout
+    });
+
+    for (const el of await page.locator(`tr >> a`).elementHandles()) {
+      const name = await el.textContent();
+
+      if (name !== testInstanceName) {
+        await deleteKafkaInstance(page, name);
+      }
+    }
+
+    if ((await page.getByText(testInstanceName).count()) === 0) {
+      await createKafkaInstance(page, testInstanceName);
+    }
+  }
+});
```

Comment:
This is something that could be generalized. We should think about it in future PRs.

Reply:
Absolutely valid point and agree.